nautobot/extras/choices.py

```diff
@@ -482,25 +482,34 @@ class SecretsGroupAccessTypeChoices(ChoiceSet):
     )
 
 
+
 class SecretsGroupSecretTypeChoices(ChoiceSet):
+    TYPE_AUTHKEY = "authentication-key"
+    TYPE_AUTHPROTOCOL = "authentication-protocol"
     TYPE_KEY = "key"
+    TYPE_NOTES = "notes"
     TYPE_PASSWORD = "password"  # noqa: S105  # hardcoded-password-string -- false positive
+    TYPE_PRIVALGORITHM = "private-algorithm"
+    TYPE_PRIVKEY = "private-key"
     TYPE_SECRET = "secret"  # noqa: S105  # hardcoded-password-string -- false positive
     TYPE_TOKEN = "token"  # noqa: S105  # hardcoded-password-string -- false positive
-    TYPE_USERNAME = "username"
     TYPE_URL = "url"
-    TYPE_NOTES = "notes"
+    TYPE_USERNAME = "username"
 
     CHOICES = (
+        (TYPE_AUTHKEY, "Authentication Key"),
+        (TYPE_AUTHPROTOCOL, "Authentication Protocol"),
         (TYPE_KEY, "Key"),
+        (TYPE_NOTES, "Notes"),
         (TYPE_PASSWORD, "Password"),
+        (TYPE_PRIVALGORITHM, "Private Algorithm"),
+        (TYPE_PRIVKEY, "Private Key"),
         (TYPE_SECRET, "Secret"),
         (TYPE_TOKEN, "Token"),
-        (TYPE_USERNAME, "Username"),
         (TYPE_URL, "URL"),
-        (TYPE_NOTES, "Notes"),
-    )
+        (TYPE_USERNAME, "Username"),
 
+    )
```

Comment:
I'm guessing this is what `ruff` is still complaining about in CI:

```suggestion
    )

```